README.md

```diff
@@ -1,2 +1,22 @@
-# sensu-plugins-vault
+## Sensu-Plugins-vault
+
+[![Build Status](https://travis-ci.org/sensu-plugins/sensu-plugins-vault.svg?branch=master)](https://travis-ci.org/sensu-plugins/sensu-plugins-vault)
+[![Gem Version](https://badge.fury.io/rb/sensu-plugins-vault.svg)](http://badge.fury.io/rb/sensu-plugins-vault)
+[![Dependency Status](https://gemnasium.com/sensu-plugins/sensu-plugins-vault.svg)](https://gemnasium.com/sensu-plugins/sensu-plugins-vault)
```

Comment:
we do not use `gemnasium` any more as it was bought by gitlab and no longer available as a standalone offering. As github gives us CVE warnings and we started using https://dependabot.com I dont think we want to include this badge anymore. I will also update the skel to reflect this.